packages/expo-speech/src/Speech/Speech.types.ts

```diff
@@ -15,6 +15,11 @@ export type SpeechOptions = {
    * Rate of the voice to speak `text`. `1.0` is the normal rate.
    */
   rate?: number;
+  /**
+   * if true, the speech follows the same behavior as `Audio.setAudioModeAsync()`
```

Comment:
```suggestion
   * If set to `true`, the speech synthesizer will use the main audio session that is usually used by other modules such as `expo-av`. Otherwise, the system creates a separate audio session to automatically manage speech, interruptions, and mixing and ducking the speech with other audio sources.
   * @default false
```